### DISCUSSION TITLE
Activate transaction for tests usin Pool

### DISCUSSION THREAD

**Discussion by dpineiden** [weiznich/diesel_async#53] [Q&A]
Hi.
I'm using the 0.2.0 version.
I have some tests to do, I pretend to activate the 'test transaction' mode with the AsyncPgConnection structure. The problem is that i have a Pool with an AsyncDieselConnection. Using 'pool.get().await.unwrap()' returns an Object that encapsulate the manager and then the pgconnection. So This is a question about how activate the 'test transactions' through this 'Object'?

Best regards!

**Comment by weiznich** [comment]
First of all: Our issue tracker is no support forum and the issue template is mandatory. As this is clearly a support question I've converted this to a discussion thread. 

That written: I'm not sure that I understand your question correctly. All methods of the `AsyncConnection` trait are callable by any pooled connection type, so I'm not sure what's your actual question. Maybe try to include some code what you are trying to do and what does not work as expected?
